benchexec/tools/depthk.py

```diff
@@ -36,17 +40,16 @@ class Tool(benchexec.tools.template.BaseTool):
 
     def executable(self):
 
-        return Util.find_executable('depthk-wrapper.sh')
+        return Util.find_executable('../../svcomp/depthk/depthk-wrapper.sh'
```

Comment:
Is this change intentional? If yes, please make a separate commit for it and explain why it is necessary. If not, please make sure *before* creating a pull request that it contains exactly the changes you want to make.